### PR TITLE
Add various security headers

### PR DESCRIPTION
**Pull request by ghickman**
Ref https://github.com/opensafely-core/interactive.opensafely.org/issues/179

### CODE REVIEW

**Comment by lucyb** [comment]
I notice this PR is to merge the branch into [remove-gateway](https://github.com/opensafely-core/output-explorer/tree/remove-gateway). Is that correct?

File: reports/settings.py [hunk 1]
```diff
@@ -180,12 +180,17 @@
 
 # CSP
 # https://django-csp.readthedocs.io/en/latest/configuration.html
+CSP_REPORT_ONLY = DEBUG
```
Comment (lucyb) [inline reports/settings.py:183]: From testing with OSI, if you want the default_src to be none, you'll need to specify it explicitly.

File: reports/settings.py [hunk 2]
```diff
@@ -180,12 +180,17 @@
...
+CSP_STYLE_SRC_ELEM = ["'self'", "https://fonts.googleapis.com"]
+CSP_SCRIPT_SRC_ELEM = ["'self'"]
```
Comment (lucyb) [inline reports/settings.py:185]: This site also uses plausible.io

**Comment by ghickman** [comment]
@lucyb – I think this one is good to go now?

**Review by lucyb — APPROVED** [review]
Security headers look fine. I don't really know much about the css/js changes, so that part of my review is more of a rubber stamp, hope that's okay

**Pull request merged**